DataDefinitions/Properties/EngineerSpecialty.resx

```diff
@@ -0,0 +1,210 @@
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AFMUs" xml:space="preserve">
+    <value>Field Maintenance Units</value>
+  </data>
+  <data name="Armour" xml:space="preserve">
+    <value>Armour</value>
+  </data>
+  <data name="Cannons" xml:space="preserve">
+    <value>Cannons</value>
+  </data>
+  <data name="ChaffAndHeatSinkLaunchers" xml:space="preserve">
+    <value>Chaff and Heat Sink Launchers</value>
+  </data>
+  <data name="ECMs" xml:space="preserve">
+    <value>Electronic Countermeasures</value>
+  </data>
+  <data name="FragCannons" xml:space="preserve">
+    <value>Frag Cannons</value>
+  </data>
+  <data name="FrameShiftDriveInterdictors" xml:space="preserve">
+    <value>Frame Shift Interdictors</value>
+  </data>
+  <data name="FrameShiftDrives" xml:space="preserve">
+    <value>Frame Shift Drives</value>
+  </data>
+  <data name="FuelScoops" xml:space="preserve">
+    <value>Fuel Scoops</value>
+  </data>
+  <data name="HullReinforcements" xml:space="preserve">
+    <value>Hull Reinforcements</value>
+  </data>
+  <data name="Lasers" xml:space="preserve">
+    <value>Lasers</value>
+  </data>
+  <data name="LifeSupportSystems" xml:space="preserve">
+    <value>Life Support Systems</value>
+  </data>
+  <data name="LimpetControllers" xml:space="preserve">
+    <value>Limpet Controllers</value>
+  </data>
+  <data name="Mines" xml:space="preserve">
+    <value>Mines</value>
+  </data>
+  <data name="Missiles" xml:space="preserve">
+    <value>Missiles</value>
+  </data>
+  <data name="MultiCannons" xml:space="preserve">
+    <value>Multi-Cannons</value>
+  </data>
+  <data name="PlasmaAccelerators" xml:space="preserve">
+    <value>Plasma Accelerators</value>
+  </data>
+  <data name="PointDefenses" xml:space="preserve">
+    <value>Point Defenses</value>
```

Comment:
I think it's an uncountable noun. Also the game uses the UK spelling, so it should be "Point Defence".

Reply:
Agreed.